PUBLISHING.rst

```diff
@@ -42,12 +42,8 @@ Once changes have been made to develop that are ready to be published, first cho
 
 Then go through the steps of merging the changes into the master branch:
 
-#. Run :code:`pytest` and make sure all the tests pass. Then run :code:`./test/cmdline_tests.sh` and make sure those tests pass.
```

Comment:
I believe the user should still run the tests themselves locally and not just rely on the CI. I'm worried people won't know to think about testing and take ownership of that otherwise. So perhaps keep this line as is? Thoughts?

Reply:
sure, I'll add it back in

Why do you think the user should still run the tests themselves? imo it's helpful to be able to run the tests locally b/c you can debug things better that way, but I'm not sure it's really necessary to run it locally again right before you push it to github where it'll get run a second time

Reply:
Because I can imagine other people in the lab not realizing that it is their job to make sure all the tests pass if they're not explicitly told. So they'll push to PR, forget about it, come back later and the tests will have failed, and then they'll go ask someone what happened, as opposed to being given immediate feedback that they wrote something which needs fixing.

For someone who knows what's going on, whatever workflow's good for them is fine. This line wouldn't be targeted at them.

Reply:
that makes sense! sounds good